### PR TITLE
show which processes get closed on daemon termination  

### PR DESCRIPTION
**Pull request by jack60612**
added to logs and RPC endpoint

### CODE REVIEW

File: chia/cmds/stop.py [hunk 1]
```diff
@@ -20,6 +21,8 @@ async def async_stop(root_path: Path, config: Dict[str, Any], group: str, stop_d
         r = await daemon.exit()
         await daemon.close()
         if r.get("data", {}).get("success", False):
+            [print(f"{service}: Stopped") for service in r["data"]["services_stopped"]]
```
Comment (mariano54) [inline chia/cmds/stop.py:24]: it will throw an error if services_stopped is not present

Comment (jack60612) [reply chia/cmds/stop.py:24]: would it not always be there?

Comment (jack60612) [reply chia/cmds/stop.py:24]: cant be too safe, ill add a check

Comment (jack60612) [reply chia/cmds/stop.py]: safe then sorry, good point

**Comment by jack60612** [comment]
squashing and rebasing for ci

**Review by paninaro — APPROVED** [review]
lgtm

**Pull request merged**